pkg/resources/rdmaPool.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2019 Mellanox All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package resources
+
+import (
+	"github.com/golang/glog"
+	"github.com/intel/sriov-network-device-plugin/pkg/types"
+	pluginapi "k8s.io/kubernetes/pkg/kubelet/apis/deviceplugin/v1beta1"
+)
+
+/*
+	rdmaResourcePool extends resourcePool and overrides:
+	GetDeviceFile(),
+	GetEnvs()
+	GetMounts()
+*/
+type rdmaResourcePool struct {
+	resourcePool
+	rdmaMount string
+}
+
+func newRdmaResourcePool(rc *types.ResourceConfig) types.ResourcePool {
+	this := &rdmaResourcePool{
+		resourcePool: resourcePool{
+			config:      rc,
+			devices:     make(map[string]*pluginapi.Device),
+			deviceFiles: make(map[string]string),
+		},
+		rdmaMount: "/dev/infiniband",
+	}
+	this.IBaseResource = this
+	return this
+}
+
+// Overrides GetDeviceFile() method
+func (rp *rdmaResourcePool) GetDeviceFile(dev string) (devFile string, err error) {
+	// There are no specific device file
+
+	return // empty string
+}
+
+func (rp *rdmaResourcePool) GetEnvs(deviceIDs []string) map[string]string {
+	glog.Infof("rdma GetEnvs() called")
+	envs := make(map[string]string)
+	values := ""
+	lastIndex := len(deviceIDs) - 1
+	for i, s := range deviceIDs {
+		values += s
+		if i == lastIndex {
+			break
+		}
+		values += ","
+	}
+	envs[rp.config.ResourceName] = values
+	return envs
+}
+
+func (rp *rdmaResourcePool) GetMounts() []*pluginapi.Mount {
+	glog.Infof("rdma GetMounts() called")
+	mounts := make([]*pluginapi.Mount, 0)
+	return mounts
+}
+
+func (rp *rdmaResourcePool) GetDeviceSpecs(deviceFiles map[string]string, deviceIDs []string) []*pluginapi.DeviceSpec {
+	glog.Infof("rdma GetDeviceSpecs() called")
+	devSpecs := make([]*pluginapi.DeviceSpec, 0)
+	// Add default common rdma device file
+	devSpecs = append(devSpecs, &pluginapi.DeviceSpec{
```

Comment:
Is there specific rdma device need to be added in devSpecs?  or is ``rdmaMount``(/dev/infiniband) all container needs to run rdma application?